src/coreclr/jit/hwintrinsiclistarm64.h

```diff
@@ -469,6 +469,7 @@ HARDWARE_INTRINSIC(AdvSimd,       StoreSelectedScalar,
 HARDWARE_INTRINSIC(AdvSimd,       StoreSelectedScalarVector64x2,                                     8,      3,      true,  {INS_st2,            INS_st2,            INS_st2,            INS_st2,            INS_st2,            INS_st2,            INS_invalid,        INS_invalid,        INS_st2,            INS_invalid},     HW_Category_MemoryStore,           HW_Flag_BaseTypeFromFirstArg|HW_Flag_HasImmediateOperand|HW_Flag_SIMDScalar|HW_Flag_SpecialCodeGen|HW_Flag_SpecialImport|HW_Flag_NeedsConsecutiveRegisters)
```

Comment:
wondering if we need `StoreSelectedScalarVector64x*` and `StoreSelectedScalarVector128x*`? Can't we know that from the tuple size of arguments and use that find out instruction?